### ISSUE TITLE
Bind error in phpmyadmin

### ISSUE THREAD

**Issue by Abhinav1217** [sprintcube/docker-compose-lamp#16]
I have followed instruction in your readme and currently this is unmodified execution of docker-compose up. Everything goes fine untill it closes down at this point. I have made sure I stopped any host processes which could conflict with it like host version of httpd, nginx, php-fpm, mariadb. I am new to docker and this is one of my first attempts to have a docker development environment.

```
Creating png-mysql ... done
Creating png-phpmyadmin ... error
Creating png-webserver  ... done

ERROR: for png-phpmyadmin  Cannot start service phpmyadmin: driver failed programming external connectivity on endpoint png-phpmyadmin (194b54bcb060f91978f51200aad67a9d3e8fd5c149f16d44c4c5ab31078309a0): Error starting userland proxy: listen tcp 0.0.0.0:8080: bind: address already in use

ERROR: for phpmyadmin  Cannot start service phpmyadmin: driver failed programming external connectivity on endpoint png-phpmyadmin (194b54bcb060f91978f51200aad67a9d3e8fd5c149f16d44c4c5ab31078309a0): Error starting userland proxy: listen tcp 0.0.0.0:8080: bind: address already in use
ERROR: Encountered errors while bringing up the project.
```
I am running fedora 29. I have used `netstat -ltnp | grep -w ':80'` and `netstat -ltnp | grep -w ':8080' ` to ensure those ports are available along with default port of php-fpm and mariadb and memcached. All those ports are available. 

*sidenote:* I actually needed this to run php5.6 so I was using that branch. But it installed php 7.2  
```
Building webserver
Step 1/26 : FROM php:7.1.20-apache
Trying to pull repository docker.io/library/php ... 
sha256:bc97986289c9397560aad8ee05ed572f88cd42d9df6aab5a0
```
I later realised that in the readme from 5.6 branch, you forgot to mentions that after doing git clone, I needed to checkout the 5.6 branch. You have mentioned it in the master branch to checkout 7.1 .  So since I didn't checkout that branch, It was installing php 7.1 . It would be helpful if you could update that readme too.

**Comment by narendravaghela** [comment]
Thanks, @Abhinav1217 for pointing out.
I have updated the readme file.

**Comment by Abhinav1217** [comment]
I figure out the problem, I had another docker process occupying the 8080 port. Just don't understand why it didn't show up on netstat command but when I pruned all the images and started from the scratch, It now worked. 

I have to say, among other docker lamp repos I tried before this, This is most simple and explanatory repo. I could actually understand enough to add a few php extensions and replace apache with nginx. Now I just need to figure out mailhog and bind integration. Thanks for making this repo.
